Integrations/GoogleBigQuery/GoogleBigQuery.yml

```diff
@@ -0,0 +1,96 @@
+description: Integration for Google BigQuery, a data warehouse for querying and analyzing
+  large databases
+detaileddescription: Use these detailed instructions in order to retrieve the API
+  key
+configuration:
+- display: Google service account JSON (a credentials JSON generated from Google API
+    Manager or from GCP console)
+  name: google_service_creds
+  defaultvalue: ""
+  type: 4
+  required: true
+script:
+  script: ''
+  type: python
+  commands:
+  - name: bigquery-query
+    arguments:
+    - name: query
+      required: true
+      description: A query string, following the BigQuery query syntax, of the query
+        to execute.
+    - name: location
+      description: The geographic location where the job should run. Required except
+        for US and EU.
+    - name: allow_large_results
+      auto: PREDEFINED
+      predefined:
+      - "True"
+      - "False"
+      description: Allow large query results tables (legacy SQL, only)
+    - name: default_dataset
+      description: A stringof the fully-qualified dataset ID in standard SQL format.
+        The value must included a project ID and dataset ID separated by dots.
+    - name: destination_table
+      description: The table where results are written. Default value is None if not
+        set.
+    - name: kms_key_name
```

Comment:
is that the name they are giving it?

Reply:
No, but it is more similar to the UI.
In the API this is called destination_encryption_configuration, but that is completely unreadable and also non descriptive.
In the UI, if you want to specify a custom encryption you need to choose one of your KMS keys to use.
So in this case I went with what's readable.
I can change it to "encryption_kms_key" or "custom_encryption_kms_key". Would that be better?